guide/english/documentation/index.md

```diff
@@ -3,14 +3,19 @@ title: Code documentation
 ---
 ## Code Documentation
 
-Code documentation is a way developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
+Code documentation is a way in which developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
 
 For example, if you happen to pass along your code to an absolute beginner, they should be able to follow along through comments, appropriate variable names and that means structuring the code in a spaced, readable manner. 
 
 It may become an extremely important idea to make a habit out of commenting your functions, loops, and declarations and treating comments as part of the source code, just as regular code should be.
 
 ![An example of commenting in the real world](https://cdn-images-1.medium.com/max/1620/1*Pyxsc7Uixbitv5myywaA_Q.jpeg)
 
+Presentation on code documentation:
+<br/><a href="http://www.youtube.com/watch?feature=player_embedded&v=pubm3spwkog
+" target="_blank"><img src="http://img.youtube.com/vi/pubm3spwkog/0.jpg" 
+alt="IMAGE ALT TEXT HERE" width="240" height="180" border="10" /></a>
```

Comment:
For **alt="IMAGE ALT TEXT HERE"**, the value of alt seems very generic. We should describe what the image is about. When the image fails to load then "IMAGE ALT TEXT HERE" won't be helpful in describing what the image is about.

One possible value  of alt attribute could be "Documentation for Better Code"